sdk/cloudmachine/Azure.CloudMachine.Web/src/CloudMachineExtensions.cs

```diff
@@ -19,6 +22,29 @@ namespace Azure.CloudMachine;
 /// </summary>
 public static class CloudMachineExtensions
 {
+    /// <summary>
+    /// Adds a CloudMachine client to the service collection.
```

Comment:
Should we comment here that `MapCloudMachineApplication` must be called first?

Reply:
Good point. I will add a comment and fix the AI generate summary above which is wrong.